sonar-dotnet-tests-library/src/main/java/org/sonar/plugins/dotnet/tests/UnitTestResults.java

```diff
@@ -50,10 +48,6 @@ public int tests() {
     return tests;
   }
 
-  double passedPercentage() {
```

Comment:
Is this related to the current ticket?

Reply:
It wasn't used so I removed it at the same time.